api/tests/integration/infrastructure/utils/pdf/certification-attestation-pdf_test.js

```diff
@@ -105,42 +105,58 @@ describe('Integration | Infrastructure | Utils | Pdf | Certification Attestation
   it('should generate a page per certificate', async function () {
```

Comment:
**question**
Sais-tu pourquoi il y a 4 tests dans un `it` ?

Reply:
On veut tester ici qu'on est capable de générer un pdf avec plusieurs certifications. C'est ce qui est fait pour le téléchargement des attestations d'une classe depuis Pix Orga.